### PR TITLE
Add static key for the `content` property in the `Post` model

### PR DESCRIPTION
**Pull request by dankorotin**
Fixes issue #35

### CODE REVIEW

File: Sources/App/Models/Post.swift [hunk 1]
```diff
@@ -57,14 +62,14 @@ extension Post: Preparation {
...
     convenience init(json: JSON) throws {
         try self.init(
-            content: json.get("content")
+            content: json.get(contentKey)
         )
     }
     
     func makeJSON() throws -> JSON {
         var json = JSON()
         try json.set("id", id)
```
Comment (tanner0101) [inline Sources/App/Models/Post.swift]: I think you can use `Post.idKey` here.

**Review by tanner0101 — COMMENTED** [review]
CircleCI needs to be fixed. Looks like `Post.` is required.

**Comment by dankorotin** [comment]
Fixed erroneous static let calls. Will pay more attention next time. And try building before pushing... 🤦‍♂️

**Review by tanner0101 — APPROVED** [review]
Thanks!

**Pull request merged**